### PR TITLE
[WIP] Dynostream sketch

### PR DESCRIPTION
**Pull request by jakepruitt**
@rclark this is just a work in progress branch to check out the parallel vs. serial setup that I have going for replication. Let me know how you want to change it!

### CODE REVIEW

**Comment by rclark** [comment]
The dynostream-replicator.js looks great. Next steps might be to streambotify this script, which should be pretty straightforward.

**Comment by jakepruitt** [comment]
Streambotified for you @rclark!

File: dynostream-replicator.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+    var q = queue();
+
+    _.forOwn(recordsById, function(records) {
+        q.defer(function(nextId) {
+            var serial = queue(1);
+
+            records.forEach(function(record) {
+                serial.defer(function(nextRecord) {
+                    console.log(record.EventID);
```
Comment (rclark) [inline dynostream-replicator.js]: Streambot provides a dedicated logger that writes logs to S3 (which can be ingested by other services). You should use that logger for any important logging, but also make sure that you're not sending excessive or duplicative log messages.

**Comment by rclark** [comment]
How about replacing the existing `index.js` with `dynostream-replicator.js` and removing `dynostream-serial.js`? I'm not sure I feel like we need to maintain the serial version, and I don't intend to maintain the kinesis-based version once streams are released. What do you think?

**Comment by jakepruitt** [comment]
Sounds good! I'm not sure how to go forward with testing, besides waiting for the release of streams. Hopefully there is a way to interact with it on the JS side. Otherwise, do you want to maybe set up testing tables in yukon?

**Comment by rclark** [comment]
Followup in #30

**Pull request closed without merging**